content/protocol-adapters/index.textile

```diff
@@ -0,0 +1,44 @@
+---
+title: Protocol Adapters
+meta_description: "Clients can use the Ably network using SSE and MQTT protocol adapters. This is especially useful where an Ably Client Library SDK is not available for your language of choice, or where platform resource constraints prohibit use of a Client Library SDK."
+meta_keywords: "SSE, MQTT, IoT, thin client, protocol adapter, lightweight clients, Arduino, Raspberry Pi, wearable"
+section: protocol-adapters
+index: 0
+jump_to:
+  Help with:
+    - MQTT#mqtt
+    - SSE#sse
+---
+
+While the recommended method for connecting to Ably is through the use of the Ably Client Library SDKs, there are certain situations where this is not possible, due to lack of platform support, or platform resource constraints. In this case, certain clients may be supported through the use of Ably protocol adapters. 
+
+Ably Client Library SDKs are the recommended method for connecting to Ably, because they offer support for a comprehensive set of Ably features, such as automatic connection management, authentication token renewal via @authUrl@, and presence, whereas the protocol adapters may not support these features: for example MQTT protocol adapter does not have support for presence, and the SSE protocol adapter does not have support for automatic token renewal (this has to be handled manually in the client code). A full list of Ably Client Library SDKs can be found on the "SDK page":https://ably.com/download, and features supported by each SDK can be found in the "features matrix":https://ably.com/download/sdk-feature-support-matrix.
```

Comment:
Would be good to link to each of these features, i.e. connection management, token, presence, etc

Reply:
Added links for each item.